cabal-install/Distribution/Client/ProjectConfig/Legacy.hs

```diff
@@ -1098,6 +1119,22 @@ legacyPackageConfigFieldDescrs =
       , "fail-when-no-test-suites", "test-wrapper" ]
   . commandOptionsToFields
   ) (testOptions' ParseArgs)
+ ++
+  ( liftFields
+      legacyBenchmarkFlags
+      (\flags conf -> conf { legacyBenchmarkFlags = flags })
+  . mapFieldNames
+      prefixBenchmark
+  . addFields
+      [ newLineListField "benchmark-options"
+          (showTokenQ . fromPathTemplate) (fmap toPathTemplate parseTokenQ)
+          benchmarkOptions
+          (\v conf -> conf { benchmarkOptions = v })
+      ]
+  . filterFields
+      [ ] -- TODO RGS: What to put here?
```

Comment:
As the comment indicates, I'm now really sure what should go here. I most copied the implementation from the corresponding code for test suites, which has this:

```hs
  . filterFields
      [ "log", "machine-log", "show-details", "keep-tix-files"
      , "fail-when-no-test-suites", "test-wrapper" ]
```

I'm not sure if any of these are relevant for `--benchmark-options`?

Reply:
`filterFields []` is equivalent to `id`, so I've decided to just delete this part. Let me know if I actually _do_ need to be filtering something for `--benchmark-options`.

Reply:
My earlier claim that "`filterFields []` is equivalent to `id`" is bogus, it seems, as there are QuickCheck tests that fail without it! I've added it back.